inc/thorvg.h

```diff
@@ -1728,9 +1728,10 @@ class TVG_API WgCanvas final : public Canvas
      * @brief Sets the drawing target for the rasterization.
      *
      * @param[in] instance WGPUInstance, context for all other wgpu objects.
-     * @param[in] surface WGPUSurface, handle to a presentable surface.
+     * @param[in] surface/texture WGPUSurface/WGPUTexture, handle to a presentable surface/texture.
```

Comment:
`Either WGPUSurface or WGPUTexture, serving as handles to a presentable surface or texture.`